### DISCUSSION TITLE
Using i18n

### DISCUSSION THREAD

**Discussion by loctran016** [vuepress-theme-hope/vuepress-theme-hope#613] [Q&A] [answered]
Hi @Mister-Hope, 
Here is my [source code](https://bitbucket.org/cokaps016/site/src) on BitBucket and here is my [site](https://efi-collection.netlify.app/) deployed using Netlify
I'm using i18n for both VietNamese and English. As you could see: I've 2 different file: `blog/article/ryzen-7-1700.md` and `blog/en/article/ryzen-7-1700.md` but both of them shown in main page `/` and english as well `/en/`. Is there anyway to fix this? Thanks

**Comment by Mister-Hope** [comment, marked as answer]
Currently the official blog plugin doesn't support i18n

But this issue will be fixed in V2 as I am going to write one.
